Server/DB/Scripts/rooms/addGuessToRoom.go

```diff
@@ -0,0 +1,51 @@
+package roomsScripts
+
+import (
+	"context"
+	"errors"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"ishtaloo.io/DB/Collections"
+	entities "ishtaloo.io/Entities"
+	wordUtils "ishtaloo.io/Utils/word"
+)
+
+func AddGuessToRoom(ctx context.Context, roomId string, guess rune) error {
+	if guess < 65 || guess > 90 {
```

Comment:
export to a seperate function in the same file labed guessIsValid(char)

Reply:
will do

Reply:
fixed